pages/clustering/high-availability.mdx

```diff
@@ -361,6 +361,9 @@ for which the timeout is used is the following:
 - FrequentHeartbeatReq -> coordinator sending to data instances
 - HeartbeatReq -> main sending to replica
 - TimestampReq -> main sending to replica
+- SystemHeartbeatReq -> main sending to replica
+- ForceResetStorageReq -> main sending to replica. The timeout is set to 60s.
+- SystemRecoveryReq -> main sending to replica. The timeout set to 5'.
```

Comment:
Maybe a dummy question, but based on linked PRs, is it SystemRecoveryReq or SystemRecoveryRpc? @as51340